data/de/rnv-motis.json

```diff
@@ -0,0 +1,272 @@
+{
+  "name": "Rhein-Neckar-Verkehr (directions.nwex.de)",
+  "attribution": {
+    "license": "https://www.govdata.de/dl-de/by-2-0",
```

Comment:
Unfortunately this license doesn't appear in either https://licenses.opendefinition.org/ nor https://spdx.org/licenses/ and it requires a link to the license source in the conditions

Reply:
oops I was wrong, its https://spdx.org/licenses/DL-DE-BY-2.0.html :)